### DISCUSSION TITLE
Connection error

### DISCUSSION THREAD

**Discussion by popa13** [bordaigorl/rmview#93] [Q&A]
Hi, I just downloading the rmview app and the library since I got a remarkable with version 2.6 and greater.

However, when I try to connect to rmview, I gMot a connection error. The error is the following:

" Please check your reMarkable is properly configured, see the documentation for instructions.
Connection failed: [Failure instance: Traceback (failure with no frames): <class
'twisted.internet.error.ConnectionRefusedError'>:
Connection was refused by other side: 111: Connection refused
] "

If you ever faced the same issue, please give me some clue to correction this connection error.
Thank you!
Pierre

**Comment by bordaigorl** [comment]
The new version should work out of the box with 2.6.
For 2.9 and above it works by using the ScreenShare feature.

**Reply by popa13** [reply]
The Screensaver feature of the Remarkable doesn't work on Linux (ubuntu) since the Remarkable app is not available in the Linux world. For example, I am using RCU to upload documents to my tablet and download documents from the tablet to replace the Remarkable app.
I would like to do the same think with Rmview

**Reply by popa13** [reply]
I probably mix up things. Is the Screensaver feature you are talking about is the one that Remarkable just added?

**Reply by popa13** [reply]
I just read #86 and #87. I will try this tomorrow. Thank you for your support.
